dev/Prototypes/ItemsViewPrototype/Control/ItemContainer.cs

```diff
@@ -220,6 +220,10 @@ private void UpdateVisualStates()
             {
                 VisualStateManager.GoToState(this, "Selected", true);
             }
+            else if (m_isPointerOver)
```

Comment:
>else if [](start = 12, length = 7)

nit: if an item is selected, then there wont be any visuals when the pointer goes over. if you want that, you will need to add a pionteroverselected state.

Reply:
I considered this, but classic file explorer does not distinguish PointerOverSelected separately either, so I figured I would leave it like this for now, until a need arises.